### PR TITLE
FSUEDOC-2433 fix context.xml file

### PR DESCRIPTION
**Pull request by MelissaFlinn**
Signed-off-by: Melissa Flinn <mflinn@redhat.com>

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [x] Did you use the Jira Issue number in the commit comments?
- [x] Did you set meaningful commit comments on each commit?
- [x] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [x] Are all Sonar reported issues fixed or can they be ignored?
- [x] Is there no duplicated code?
- [x] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [x] Have you used the correct file header copyright comment?
- [x] Have you used the correct year in the headers of new files?

### CODE REVIEW

**Comment by MelissaFlinn** [comment]
I checked with Fintan - the commented entries were done manually (not automatically)
Entries are commented out because they are not in the Camel component reference because:
- the component is NOT supported in Fuse
OR
- the support status of the component is unclear or uncertain
There might be a few excluded entries that ought to have been included but we are erring on the safe side by leaving them out.

**Comment by apupier** [comment]
removing "PRIO", this is a "bugfix" as same level than other bugs. Thanks to previous merged PR,we are in the same state level than for 10.2 (and with updated content)

File: core/plugins/org.fusesource.ide.help/contexts.xml [hunk 1]
```diff
@@ -259,168 +250,254 @@
...
+    <context id="azure-blob">
+        <description>Azure Storage Blob Service Component Options</description>
+        <topic href="azure-blob-component.html" label="Azure Storage Blob Service"/>
+    </context>
+    <context id="azure-queue">
+        <description>Azure Storage Queue Service Component Options</description>
+        <topic href="azure-queue-component.html" label="Azure Storage Queue Service"/>
     </context>
-    <context id="beanEIP">
+    <context id="bean">
```
Comment (apupier) [inline core/plugins/org.fusesource.ide.help/contexts.xml]: why there is no more a beanEIP? When selecting the "Bean" on the diagram, it is currently searching for a "beanEIP" context ID. So cannot find it anymore, and the documentation page displayed is then the generic one:
image

Comment (MelissaFlinn) [reply core/plugins/org.fusesource.ide.help/contexts.xml]: I can fix this. I can update the context.html to add a context ID = "beanEIP" and point to the bean.html file. Should I just commit it to this same branch for this pull request?

Comment (apupier) [reply core/plugins/org.fusesource.ide.help/contexts.xml]: Yes, on same branch is perfect

**Pull request merged**